docs/reference/NuGet-Client-SDK.md

```diff
@@ -9,14 +9,66 @@ ms.topic: conceptual
 
 # NuGet Client SDK
 
-The *NuGet Client SDK* refers to a group of NuGet packages centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the earlier [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library.
+The *NuGet Client SDK* refers to a group of NuGet packages centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the legacy [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library. You can find the source code for these packages on GitHub in the project [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client).
 
 > [!Note]
 >  For documentation on the NuGet server protocol, please refer to the [NuGet Server API](~/api/overview.md).
 
-## Source code
+## Getting started
 
-The source code is published on GitHub in the project [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client).
+### Install the package
+
+```
+dotnet add NuGet.Protocol
+```
+
+## Examples
+
+### List package versions
+
+```csharp
+Repository.Factory.GetCoreV3("https://api.nuget.org/v3/index.json");
+
+
+```
+
+### Download a package
+
+```csharp
+var foo = "bar";
+```
+
+### Get package metadata 
+
+```csharp
+var foo = "bar";
+```
+
+### Search packages
+
+Search for "json" packages using the [NuGet V3 Search API](../api/search-query-service-resource.md):
+
+```csharp
+var logger = NullLogger.Instance;
+var cancellationToken = CancellationToken.None;
+
+var repository = Repository.Factory.GetCoreV3("https://api.nuget.org/v3/index.json");
+var search = await repository.GetResourceAsync<PackageSearchResource>();
+var searchFilter = new SearchFilter(includePrerelease: true);
+
+var results = await search.SearchAsync(
+    "json",
+    searchFilter,
+    skip: 0,
+    take: 20,
+    logger,
+    cancellationToken);
+
+foreach (var result in results)
+{
+    Console.WriteLine($"Found package {result.Identity.Id} {result.Identity.Version}");
+}
+```
 
 ## Third-party documentation
```

Comment:
TODO: `Martin Björkström did a follow-up blog post to Dave Glick's blog series where he introduces a different approach on using the NuGet Client SDK for installing NuGet packages:` -> `Martin Björkström did a follow-up blog post to Dave Glick's blog series where he introduces a different approach on using the NuGet Client SDK to install NuGet packages:`